Editor/CoreAPI/StateManager.cs

```diff
@@ -51,5 +58,12 @@ private void SetProfile(string profileName)
             ComputeManager = new GameLiftComputeManager(CoreApi, GameLiftWrapper);
             OnProfileSelected?.Invoke();
```

Comment:
Modify to OnUserProfileUpdated and reuse in the SetBucketBootstrap method

Reply:
updated